### DISCUSSION TITLE
How to attract interviewers?

### DISCUSSION THREAD

**Discussion by MaksymMatveichuk** [ratifire/devrate-service#405] [Ideas]
For example, we can give reviewers the opportunity to provide individual or group lessons for a fee.

**Comment by BlackPuma9** [comment]
Good to have ranking of Top 20 reviewers that would motivate people to provide interview. And each week, for example, Monday morning, each user would receive push notification with that ranking.
On top of that idea we can implement rewards system and some kinda label that would be visible in profile page.
